src/components/Book.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ id, title, category }) => (
+  <tr>
+    <td>{id}</td>
+    <td>{title}</td>
+    <td>{category}</td>
+  </tr>
+);
+
+Book.propTypes = {
+  id: PropTypes.number,
+  title: PropTypes.string,
+  category: PropTypes.string,
+};
```

Comment:
- [ ] Given the context of the application it makes more sense to make all these props required since we always need this information to create a new book.
```JS
Book.propTypes = {
  id: PropTypes.number.isRequired,
  title: PropTypes.string.isRequired,
  category: PropTypes.string.isRequired,
};
```